frame/staking-rewards/src/test/runtime.rs

```diff
@@ -200,5 +216,13 @@ impl pallet_staking_rewards::Config for Runtime {
 	type UnixTime = Timestamp;
```

Comment:
so you can use UnixTime trait to check block I guess, not direct deps on other pallet.

Reply:
in Lending we switched from Blocks to Seconds afaik for some reason were Karol was in discussion.

Reply:
I'm actually thinking we could remove this `end_block` field as it creates more trouble than help and probably irrelevant to the overall design as originally intended. cc @KaiserKarel